docs/APIReference-Route.md

```diff
@@ -70,6 +76,72 @@ class ProfileRoute extends Relay.Route {
 }
 ```
 
+### prepareParams (static property)
+
+```
+static prepareParams: ?(prevParams: {[prevParam: string]: mixed}) => {[param: string]: mixed};
+```
+
+Routes can declare additional parameters, or conversion for parameters on initialisation. These can also be used as query parameters in your nested containers.
+
+#### Example
+
+```
+class ProfileRoute extends Relay.Route {
+  static queries = {
+    viewer: () => Relay.QL`query { viewer }`
+  };
+  static prepareParams = (prevParams) => {
+    // Make sure any params that ProfileRoute gets initialised with will still be passed down.
+    return {
+      ...prevParams,
+      limit: 10
```

Comment:
to help clarify the other comment, it might be good to comment this line too, maybe something like "and override the `limit` variable in the top level container.